widget-reference/chart.md

```diff
@@ -61,6 +63,17 @@ description: >-
       <td style="text-align:left">Controls widget&apos;s visibility on the page. When turned off, the widget
         will not be visible when the app is published</td>
     </tr>
+    <tr>
+      <td style="text-align:left"><b>Add chart</b>
```

Comment:
This is not a separate property of the chart and should be a note of the chart data property

Reply:
@Nikhil-Nandagopal  I guess I fixed all the issues 👍